### PR TITLE
!MERGE - Improve bootstrap output

### PR DESCRIPTION
**Pull request by thomasdigby**
Any ideas for the default API that comes with bootstrappin? Also, much to do to this, but comments and discussion welcome

### CODE REVIEW

**Comment by jonsherrard** [comment]
Difficult one, we probably need permission. Is there an official WordPress.org blog perhaps...

I'll have a look around.

**Comment by jonsherrard** [comment]
I think this could be the one!

https://wordpress.org/news/wp-json/wp/v2/posts

So `https://wordpress.org/news`

**Comment by mathewhawley** [comment]
What do you mean exactly by the 'bootstrap API'? Users/parent projects can do something to bootstrap a project with certain settings etc?

**Comment by thomasdigby** [comment]
Ah no, just the default `siteUrl` in the `tapestry.config`

File: src/shared/default-routes.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ export default ({
     allowEmptyResponse: true
   }
 }, {
-  path: '/:year/:monthnum/:day/:postname',
+  path: '/:category/:year/:monthnum/:postname',
```
Comment (jonsherrard) [inline src/shared/default-routes.js]: Presuming this change is for compatibility with `wordpress.org/news`, but the actual defaults for wordpress are as above. Could add a custom route into the bootstrapped project @thomasdigby ?

Comment (thomasdigby) [inline src/shared/default-routes.js]: Ah I didn't realise it was the Wordpress default, I'll revert it. It'd be nice to rely on the default Tapestry routes, lets chat more about it tomorrow, I think this task could do with some short term/long term definition.

**Comment by thomasdigby** [comment]
OHHHHHHHH MEEEEEERRRRRCCCCCCYYYYYYYYY

**Pull request merged**